module-2/backend/week_two.md

```diff
@@ -5,16 +5,37 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  - Active Record is a database language that allows to create tables, records, and database objects that integrate with our Ruby programs. It provides us with methods and commands that allows us to relate and use these objects to display and manipulate data.
+
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  - I believe they are instance methods that form the relationship of the objects in our programs.
+
 3. What do they allow you to do?
+  - belongs_to and has_many builds the relationship between DB objects and access those objects through separate but interconnected tables.
```

Comment:
it builds the relationship between our models, not our database. we still need to create that relationship on our database level 👍